src/components/CodeModal.tsx

```diff
@@ -7,35 +7,33 @@ import {
   ModalHeader,
   ModalOverlay,
 } from "@chakra-ui/react"
-export interface IProps {
-  children?: React.ReactNode
+
+type CodeModalProps = {
+  title: string
+  children: ReactNode
   isOpen: boolean
   setIsOpen: (isOpen: boolean) => void
-  title: string
 }
 
-const CodeModal: React.FC<IProps> = ({
-  children,
-  isOpen,
-  setIsOpen,
-  title,
-}) => {
-  return (
-    <Modal
-      isOpen={isOpen}
-      scrollBehavior="inside"
-      variant="code"
-      onClose={() => setIsOpen(false)}
-    >
-      <ModalOverlay />
-      <ModalContent>
-        <ModalHeader>{title}</ModalHeader>
-        <ModalCloseButton />
-
-        <ModalBody>{children}</ModalBody>
-      </ModalContent>
-    </Modal>
-  )
-}
+const CodeModal = ({ children, isOpen, setIsOpen, title }: CodeModalProps) => (
+  <Modal
+    isOpen={isOpen}
+    scrollBehavior="inside"
+    variant="code"
+    onClose={() => setIsOpen(false)}
+  >
+    <ModalOverlay />
+    <ModalContent>
+      <ModalHeader>{title}</ModalHeader>
+      <ModalCloseButton
+        style={{
+          right: "unset",
+          insetInlineEnd: "var(--eth-sizes-4)",
+        }}
```

Comment:
This is a chakra component. Why not use style props?
```suggestion
        right="unset"
        insetInlineEnd="4"
```

Reply:
Tried this to start but I'm realizing that the `insetInlineEnd` style prop does not seem to be properly recognized by our Chakra components... this has let to a handful of bugs throughout the site where we replaced "left" or "right" with these style props and are not being recognized

For example, https://staging--ethereumorg.netlify.app/ar/roadmap If you go to our current staging branch, this page is set to `rtl` but our use of these style props in the hero are not working, which is why the Title box is still on the left.

I have another PR open on the canonical repo to address some of these:
https://github.com/ethereum/ethereum-org-website/pull/11905/files

Not 100% sure why these specific style props don't seem to be recognized... @TylerAPfledderer Any idea on this one?

Reply:
We may be able to get away with `right="unset"`, but the other I've found only works if I nest inside `style`.

And small alternative as well if we want to make it a little more Chakra-native, could use `const insetInlineEnd = useToken("space", "4")` and then `style={{ insetInlineEnd }}`

Reply:
Ah okok, no worries then. Didn't know about that problem 🤔